### PR TITLE
Add go_package Annotation to generated protobufs.

### PR DESCRIPTION
**Pull request by robshakir**
`protoc-gen-go` now requires an explicit `go_package` annotation to specify the name of the package that is being generated. This change adds this to protobuf packages that are generated by ygot.

A new `go_package_base` option is added to the `proto_generator` binary and the name of the package put into generated packages.

### CODE REVIEW

File: proto_generator/protogenerator.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ var (
 	skipEnumDedup                        = flag.Bool("skip_enum_deduplication", false, "If set to true, all leaves of type enumeration will have a unique enum output for them, rather than sharing a common type (default behaviour).")
 	useDefiningModuleForTypedefEnumNames = flag.Bool("typedef_enum_with_defmod", false, "If set to true, all typedefs of type enumeration or identity will be prefixed with the name of its module of definition instead of its residing module.")
 	useConsistentNamesForProtoUnionEnums = flag.Bool("consistent_union_enum_names", false, "If set to true, uses more consistent names for enumerations defined within a union. This flag is only active when typedef_enum_with_defmod is also set.")
+	goPackageBase                        = flag.String("go_package_base", "", "Base name for the Go packages that are to generated - this value is included in the go_package option of the generated protobufs - and has generated packages' names appended to it.")
```
Comment (wenovus) [inline proto_generator/protogenerator.go:56]: typo nit: are to **be** generated

File: ygen/protogen.go [hunk 2]
```diff
@@ -183,6 +184,11 @@ import "{{ .YwrapperPath }}/ywrapper.proto";
 import "{{ .YextPath }}/yext.proto";
 {{- range $importedProto := .Imports }}
 import "{{ $importedProto }}";
+{{- end -}}
+
+{{- if ne .GoPackageName "" }}
```
Comment (wenovus) [inline ygen/protogen.go:189]: nit: `{{- if .GoPackageName }}` is sufficient.

**Review by wenovus — APPROVED** [review]

File: proto_generator/protogenerator.go (hunk 1 above)
Comment (robshakir) [reply proto_generator/protogenerator.go:56]: Fixed, thanks.

File: ygen/protogen.go (hunk 2 above)
Comment (robshakir) [reply ygen/protogen.go]: Thanks, fixed.

**Pull request merged**